spec/models/adventure_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe Adventure do
+  subject { FactoryGirl.build(:adventure) }
+  it { should belong_to(:bucket_list) }
+
+  it { should have_valid(:name).when('Visit Sue', 'Eat alligator') } ||
+  it { should have_valid(:address).when('Paris, France', '33 Harrison Ave, Boston') }
```

Comment:
Line is too long. [85/80]<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.